setup_local.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+source /t2k/hyperk/software/hyperk-releases/v1.3.5/hk-hyperk/Source_At_Start.sh; source /t2k/hyperk/software/wcsim-head/build/bin/this_wcsim.sh;
```

Comment:
Please no paths on specific clusters!

We want to push clusters, but this could be useful for some people who are slow to move in that direction. So what you could do is have a check for `$WCSIM_BUILD_DIR`
- if it exists, carry on
- if it doesn't, exit with a "please setup the HK software stack including WCSim, Geant, ROOT" message